AEPMessaging/Sources/Messaging+EdgeEvents.swift

```diff
@@ -151,13 +151,99 @@ extension Messaging {
+    ///
+    /// - Parameters:
+    ///   - liveActivityID: The unique identifier for the Live Activity instance associated with the update token.
+    ///   - token: The Live Activity push update token.
+    ///   - event: The original `Event` that triggered the request to send the update token.
+    func sendLiveActivityUpdateToken(liveActivityID: String, token: String, event: Event) {
+        let liveActivityData: [String: Any?] = [
+            MessagingConstants.XDM.Push.LIVE_ACTIVITY_ID: liveActivityID,
+            MessagingConstants.XDM.Push.TOKEN: token
+        ]
+
+        // Creating Edge event data with XDM and data payloads
+        let xdmEventData: [String: Any] = [
+            MessagingConstants.XDM.Key.XDM: [
+                MessagingConstants.XDM.Key.EVENT_TYPE: MessagingConstants.XDM.Push.EventType.LIVE_ACTIVITY_UPDATE_TOKEN
+            ],
+            MessagingConstants.XDM.Key.DATA: liveActivityData
+        ]
+
+        let pushTokenEdgeEvent = event.createChainedEvent(
+            name: MessagingConstants.Event.Name.LIVE_ACTIVITY_UPDATE_TOKEN_EDGE,
+            type: EventType.edge,
+            source: EventSource.requestContent,
             data: xdmEventData
         )
         dispatch(event: pushTokenEdgeEvent)
     }
 
+    /// Sends an Edge request event containing a Live Activity start event using a broadcast channel ID.
+    ///
+    /// - Parameters:
+    ///   - channelID: The unique identifier for the Live Activity broadcast channel.
+    ///   - event: The original `Event` that requested tracking the start of the Live Activity.
+    func sendLiveActivityStart(channelID: String?, origin: String, event: Event) {
+        sendLiveActivityStart(channelID: channelID, liveActivityID: nil, origin: origin, event: event)
+    }
+
+    /// Sends an Edge request event containing a Live Activity start event using a Live Activity ID.
+    ///
+    /// - Parameters:
+    ///   - liveActivityID: The unique identifier for the Live Activity.
+    ///   - event: The original `Event` that requested tracking the start of the Live Activity.
+    func sendLiveActivityStart(liveActivityID: String?, origin: String, event: Event) {
+        sendLiveActivityStart(channelID: nil, liveActivityID: liveActivityID, origin: origin, event: event)
+    }
+
     // MARK: - private methods
 
+    /// Sends an Edge request event containing a Live Activity start event using the provided broadcast channel ID or Live Activity ID.
+    /// The channel ID and Live Activity ID are mutually exclusive. Only one should be provided per call.
+    ///
+    /// - Parameters:
+    ///   - channelID: The unique identifier for the Live Activity broadcast channel.
+    ///   - liveActivityID: The unique identifier for the Live Activity.
+    ///   - event: The original `Event` that requested tracking the start of the Live Activity.
+    private func sendLiveActivityStart(channelID: String?, liveActivityID: String?, origin: String, event: Event) {
```

Comment:
i think i'd prefer we give default values of nil for `channelID` and `liveActivityID` here so we don't need those helper methods above.  then you can either do: 
```
sendLiveActivityStart(channelID: channelID, origin: origin, event: event)
== OR ==
sendLiveActivityStart(liveActivityID: liveActivityId, origin: origin, event: event)
```
depending on your need